src/frontend/get-github-configuration.ts

```diff
@@ -90,6 +90,8 @@ const getInstallationsWithAdmin = async (
 
 const removeFailedConnectionsFromDb = async (req: Request, installations: InstallationResults, jiraHost: string): Promise<void> => {
 	await Promise.all(installations.rejected
+		// Only uninstall deleted installations
+		.filter(failedInstallation => failedInstallation.deleted)
```

Comment:
That's an important change. Was it caught by tests? (If not, should we add some tests?)